Cargo.toml

```diff
@@ -26,12 +26,12 @@ env_logger = "0.3"
 filetime = "0.1"
 flate2 = "0.2"
 fs2 = "0.2"
-git2 = "0.4"
-git2-curl = "0.5"
+git2 = "0.5"
```

Comment:
Ah FWIW I had to end up yanking the 0.5 release that had support for openssl 1.1, so this'll need to be updated to 0.6 now